patches/chrome-browser-extensions-api-settings_private-prefs_util.cc.patch

```diff
@@ -1,15 +0,0 @@
-diff --git a/chrome/browser/extensions/api/settings_private/prefs_util.cc b/chrome/browser/extensions/api/settings_private/prefs_util.cc
```

Comment:
These pref keys are being whitelisted from the javascript `settingsPrivate` API. Moved from patch to override, anything extra need testing?